checkov/main.py

```diff
@@ -585,6 +585,11 @@ def add_parser_args(parser: ArgumentParser) -> None:
     parser.add('--summary-position', default='top', choices=SUMMARY_POSITIONS,
                help='Chose whether the summary will be appended on top (before the checks results) or on bottom '
                     '(after check results), default is on top.')
+    parser.add('--skip-resources-without-violations',
+               help="exclude extra resources (resources without violations) from report output",
+               default=os.environ.get('SKIP_RESOURCES_WITHOUT_VIOLATIONS', False),
+               action='store_true',
+               env_var='SKIP_RESOURCES_WITHOUT_VIOLATIONS')
```

Comment:
```suggestion
               default=False,
               action='store_true',
               env_var='SKIP_RESOURCES_WITHOUT_VIOLATIONS')
```
we try to prefix user faced env vars with `CKV_` and I think you don't need to check for the env var in the default line.